### DISCUSSION TITLE
Embind bindings in side modules?

### DISCUSSION THREAD

**Discussion by alienself** [emscripten-core/emscripten#19087] [General]
Hi,

Is it possible to create bindings using embind in emscripten inside side modules since by default they do not generate a javascript file?
Would it somehow be possible to generate a small js file for them that can be loaded before loading the side module and that would add bindings for that module?

**Comment by sbc100** [comment]
embind itself doesn't generate JS at build time, but instead creates stuff during startup when the EMSCRIIPTEN_BINDINGS block is executed.    Aside from some open issues I think it should just work in side module.  Have you tried it?
